keps/sig-windows/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.swp
+*~
```

Comment:
seems like probably this should be in the root .gitignore? :-)

Reply:
moved into https://github.com/kubernetes/enhancements/pull/702